app/models/github_repository.rb

```diff
@@ -228,6 +230,12 @@ def temp_clone_url
     end
   end
 
+  def template?
+    options = { accept: TEMPLATE_REPOS_API_PREVIEW }
+    endpoint_url = "#{GITHUB_API_HOST}/repositories/#{@id}"
```

Comment:
Why not rather use:
```suggestion
    endpoint_url = "#{StarterCodeImportable::GITHUB_API_HOST}/repositories/#{@id}"
```

Reply:
Good idea. I'll actually do the reverse and keep these constants in `github_repository.rb` since `StarterCodeImportable` doesn't use those constants.